More Documentation (#20)
srobo/astoria: docs/implementation/managers/index.rst

State Managers
==============

.. toctree::
   :maxdepth: 1
   :caption: List of State Managers:

   astdiskd
   astmetad
   astprocd

A state manager is a process that stores and mutates some state.

> -  A state manager must publish it’s state to MQTT when the state

Reviewer: ```suggestion
-  A state manager must publish its state to MQTT when the state
```
